test/smoke/hsa-lazy-queues/Makefile

```diff
@@ -5,17 +5,21 @@ TESTSRC_MAIN = async-events.cpp
 TESTSRC_AUX  = empty-sink.c
 TESTSRC_ALL  = $(TESTSRC_MAIN) $(TESTSRC_AUX)
 
-AOMPROCM ?= $(AOMP)
+AOMPROCM     ?= $(AOMP)
 ifneq (,$(wildcard $(AOMPROCM)/bin/rocprof))
 	RUNPROF   = $(AOMPROCM)/bin/rocprof
 else
 	RUNPROF   = $(AOMPROCM)/../bin/rocprof
 endif
 
+ifeq ($(ROCR_VISIBLE_DEVICES),)
```

Comment:
I just realized that this does not guard against the case where a user would do `ROCR_VISIBLE_DEVICES=2,3`.
That would still let the test case fail. Is there something we can (easily) do to mitigate that error mode?

Reply:
Ok, what should the value of `ROCR_VISIBLE_DEVICES` be in the case where a user had provided `2,3`?

If I understand your concern correctly we should override in that case and only make one device visible to the test regardless of the environmental setting.
It that is preferred, I'd split on ',' and take the first (or last) 'result' as the visible device override.

Reply:
yeah that's a good idea. I don't know for sure how real-world that concern even is.
Thanks for providing this fix.